### DISCUSSION TITLE
Stamp DNS with dns server address

### DISCUSSION THREAD

**Discussion by Ninjanoir78** [DNSCrypt/dnscrypt-proxy#2240] [Q&A]
Hi, 

I try to use the new dns server, control D,   I try to setup the dns with my dnscrypt, I use it as basic but I try to use with doh/3 with custom filters, here below what their website gives me :

https://dns.controld.com/8htsokatxi
and from there, I try to create a stamp from https://dnscrypt.info/stamps/
but not sure if it is possible ?
thanks

**Comment by jedisct1** [comment]
You can use a static entry: https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Configuration#an-example-static-server-entry

**Reply by jedisct1** [reply]
I guess for that server the stamp would be 
    sdns://AgMAAAAAAAAAACCaOjT3J965vKUQA9nOnDn48n3ZxSQpAcK6saROY1oCGRBkbnMuY29udHJvbGQuY29tCy84aHRzb2thdHhp
